project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
 addSbtPlugin("com.twitter" % "scrooge-sbt-plugin" % "22.1.0")
 
-addSbtPlugin("com.gu" % "sbt-scrooge-typescript" % "2.0.0")
+addSbtPlugin("com.gu" % "sbt-scrooge-typescript" % "4.0.0-PREVIEW.fpdowngrade-scrooge.2025-10-02T0939.1433794d")
```

Comment:
Just need to update this once https://github.com/guardian/scrooge-extras/pull/39 is merged and released, so that we can use the non-preview version here!